<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dev-AjmalSyed" failures="2" tests="4" name="Positions_Validations" time="77.133" errors="0" timestamp="2023-11-24T17:44:45 PKT">
  <testcase classname="IdentityServer_API_User.Login" name="UserLogin" time="3.17"/>
  <testcase classname="Subscriptions_V3.Order_Positions_Subscribe" name="Verify_Subscribe_BUY_Order_Positions" time="24.66"/>
  <testcase classname="Subscriptions_V3.Order_Positions_Subscribe" name="Verify_Subscribe_SELL_Order_Positions" time="23.727">
    <failure type="java.lang.AssertionError" message="Validate_completeDayBuyOrderQty expected [30.0] but found [20.0]">
      <![CDATA[java.lang.AssertionError: Validate_completeDayBuyOrderQty expected [30.0] but found [20.0]
at APIHelper.APIHelperClass.Validate_Positions(APIHelperClass.java:1280)
at Subscriptions_V3.Order_Positions_Subscribe.Verify_Subscribe_SELL_Order_Positions(Order_Positions_Subscribe.java:643)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Subscribe_SELL_Order_Positions -->
  <testcase classname="Subscriptions_V3.Order_Positions_Subscribe" name="Verify_Subscribe_SHORTSELL_Order_Positions" time="21.155">
    <failure type="java.lang.AssertionError" message="Validate_completeDaySellLongOrderQty expected [25.0] but found [5.0]">
      <![CDATA[java.lang.AssertionError: Validate_completeDaySellLongOrderQty expected [25.0] but found [5.0]
at APIHelper.APIHelperClass.Validate_Positions(APIHelperClass.java:1281)
at Subscriptions_V3.Order_Positions_Subscribe.Verify_Subscribe_SHORTSELL_Order_Positions(Order_Positions_Subscribe.java:890)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Subscribe_SHORTSELL_Order_Positions -->
</testsuite> <!-- Positions_Validations -->
